config/locales/en/course/video/videos.yml

```diff
@@ -27,13 +27,15 @@ en:
         destroy:
           success: '%{title} was deleted.'
           failure: 'Could not delete video: %{error}'
+        todo_video_button:
```

Comment:
I think you need to update this test to expect this translation

https://github.com/Coursemology/coursemology2/blob/023de7daf90932478b74678ff09aaa6dc75dcb23/spec/features/course/homepage_spec.rb#L174